lib/v2/cgtn/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (router) => {
+    router.get('/podcast/:category/:id/:content?', require('./podcast.js'));
```

Comment:
```suggestion
    router.get('/podcast/:category/:id', require('./podcast.js'));
```
Unused parameter `ctx.params.content`